common/core/package.json

```diff
@@ -14,25 +14,57 @@
     "@types/getos": "^3.0.0",
     "@types/node": "^9.6.50",
     "chai": "^4.2.0",
-    "istanbul": "^0.4.5",
     "mocha": "^6.1.4",
+    "nyc": "^15.0.0",
     "sinon": "^7.4.1",
+    "source-map-support": "^0.5.16",
     "tslint": "^5.18.0",
     "typescript": "2.9.2"
   },
   "scripts": {
     "lint": "tslint --project . -c ../../tslint.json",
     "build": "tsc",
-    "unittest-min": "istanbul cover --report none ../../node_modules/mocha/bin/_mocha -- --reporter dot test/_*_test.js",
-    "alltest-min": "istanbul cover ../../node_modules/mocha/bin/_mocha -- --reporter dot test/_*_test*.js",
-    "unittest": "istanbul cover ../../node_modules/mocha/bin/_mocha -- --reporter spec test/_*_test.js",
-    "alltest": "istanbul cover ../../node_modules/mocha/bin/_mocha -- --reporter spec test/_*_test*.js",
-    "ci": "npm -s run lint && npm -s run build && npm -s run alltest-min && npm -s run check-cover",
-    "test": "npm -s run lint && npm -s run build && npm -s run unittest",
-    "check-cover": "istanbul check-coverage --statements 98 --branches 96 --functions 98 --lines 99"
+    "unittest-min": "nyc --reporter=lcov --report none ../../node_modules/mocha/bin/_mocha -- dot test/_*_test.js",
+    "alltest-min": "nyc --reporter=lcov ../../node_modules/mocha/bin/_mocha -- dot test/_*_test*.js",
+    "unittest": "nyc --reporter=lcov --reporter=text ../../node_modules/mocha/bin/_mocha",
+    "alltest": "nyc --reporter=lcov --reporter=text ../../node_modules/mocha/bin/_mocha -- test/_*_test*.js",
+    "ci": "npm -s run lint && npm -s run build && npm -s run alltest-min",
+    "test": "npm -s run lint && npm -s run build && npm -s run unittest"
+  },
+  "nyc": {
+    "exclude": [
+    	"coverage/**",
+      "packages/*/test{,s}/**",
+      "**/*.d.ts",
+      "test{,s}/**",
+      "test{,-*}.{js,cjs,mjs,ts}",
+      "**/*{.,-}test.{js,cjs,mjs,ts}",
+      "**/__tests__/**",
+
+      "**/{ava,nyc}.config.{js,cjs,mjs}",
+      "**/jest.config.{js,cjs,mjs,ts}",
+      "**/{karma,rollup,webpack}.config.js",
+      "**/{babel.config,.eslintrc,.mocharc}.{js,cjs}",
+
+      "common.js",
+      "src/authentication_provider.ts",
+      "src/results.ts"
+    ],
+    "extension": [
+      ".ts",
+      ".tsx"
+    ],
+    "all": true,
+    "source-map": true,
+    "produce-source-map": true,
+    "check-coverage": true,
+    "statements": 99,
+    "lines": 99,
+    "functions": 99,
+    "branches": 80
```

Comment:
> 80 [](start = 15, length = 3)

Why did this go down?

Reply:
@anthonyvercolano it's unclear why we initially had branches at 96... I am not changing any code so my guess is maybe a bug in previous versions of NYC. 

Additionally, we're now doing testing not just of the generated javascript but mapping that to the lines in typescript, so that will affect the number of lines that are being registered as tested.